### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by CleanMachine1**

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Also don't you mean x86 rather than x64..

**Comment by Itai-Nelken** [comment]
I do mean x64 (or amd64), x86 is i386 (i686). github actions only support 64bit x86 (unless you use qemu and docker stuff).
if you think that x86_64 or x86 (64 bit) sounds better you can change it.

**Comment by CleanMachine1** [comment]
amd64 wiki, type that here 

duck.com
or your spying google and you will find that amd64 refers to  X86-64

**Comment by CleanMachine1** [comment]
I will update it in a few hours

**Comment by Itai-Nelken** [comment]
I edited the builds and checks, now they are in the same workflow, and I got arm64 working on github actions! there is a problem with cmake on armhf docker containers running with qemu... so my self hosted runner will be for armhf for now...

**Comment by CleanMachine1** [comment]
How is it self hosted, how is your pi communicating with git and github?

File: README.md [hunk 1]
```diff
@@ -97,10 +97,10 @@ source .bashrc
 
 ## Daily builds
 
-Daily builds for x64 and armhf are available in the 'actions' tab of this repo, click on 'build-release' or 'build-release-armhf',
+Daily builds for x64 and armhf are now available, these can be found in the actions tab on the aptpac repo, once you have clicked on the actions tab, click on 'build-release' or 'build-release-armhf' then
```
Comment (Itai-Nelken) [inline README.md:100]:
```suggestion
Daily builds for x64, arm64, and armhf are now available, these can be found in the actions tab on the aptpac repo, once you have clicked on the actions tab, click on 'Build-x64', 'Build-arm64', or 'Build-armhf' then
```

File: README.md [hunk 2]
```diff
@@ -97,10 +97,10 @@ source .bashrc
...
 click on the latest build with a green ✔️, scroll down to the 'artifacts' section, and click on the zip there.
 keep in mind you need a github account to download them.
 
 ## Build checks
 
-Build checks for arm64, armhf, and x64 are run for every commit and pr. you can see the status in the badges at the top of the readme.
+Build checks for ARM64, ARMhf, and X86-64 are run for every commit and pr. you can see the status in the badges at the top of the README.
```
Comment (Itai-Nelken) [inline README.md:106]: personal preference, but I prefer 'armhf' and 'arm64' (lowercase)

**Comment by Itai-Nelken** [comment]
> How is it self hosted, how is your pi communicating with git and github?

github support armhf runners, so you just have top create one in the settings of the repository and they provide you with the commands to install the software.

File: README.md (hunk 2 above)
Comment (CleanMachine1) [reply README.md:106]: it is actually meant to be in caps, and I think it actually looks better, however you choose!

**Comment by CleanMachine1** [comment]
> > How is it self hosted, how is your pi communicating with git and github?
> 
> github support armhf runners, so you just have to create one in the settings of the repository and they provide you with the commands to install the software.

very interesting, have you got a link or anything to read up on

**Comment by Itai-Nelken** [comment]
https://docs.github.com/en/actions/hosting-your-own-runners/adding-self-hosted-runners
I know it isn't very secure to use on public repositories... thats why I only start it when I need it to run the build checks.

**Comment by Itai-Nelken** [comment]
I don't think it uses actions minutes (I hope)

File: README.md (hunk 2 above)
Comment (Itai-Nelken) [reply README.md:106]: I prefer lowercase

**Comment by CleanMachine1** [comment]
About it being unsafe, surely you should edit the workflow files to remove it running at PR, and just have it as commits maybe, or something where only running the action when code is approved

**Review by CleanMachine1 — COMMENTED** [review]
This commit is fine

**Comment by CleanMachine1** [comment]
Don't know what happened with that commit, it seem to bug out the logos because of a line I had put when linting

**Comment by Itai-Nelken** [comment]
I want it to run on pr's, but anyway I only run it when I need it and after I review the code and make sure its not malicious. if you look at how much time the first armhf checks for this pr took, you will see its ~8 hours, that is because it had to wait until I ran the script.

**Comment by Itai-Nelken** [comment]
> Don't know what happened with that commit, it seem to bug out the logos because of a line I had put when linting

I fixed them.

**Comment by Itai-Nelken** [comment]
can I merge?

**Comment by Itai-Nelken** [comment]
thanks a lot!

**Pull request merged**